modules/tfm/tfm/boards/src/tfm_platform_system.c

```diff
@@ -39,6 +39,8 @@ tfm_platform_hal_fw_info_service(psa_invec  *in_vec, psa_outvec *out_vec)
 	uint32_t attr = TFM_HAL_ACCESS_WRITABLE |
 			TFM_HAL_ACCESS_READABLE |
 			TFM_HAL_ACCESS_NS;
+	uintptr_t boundary = (1 << HANDLE_ATTR_NS_POS) &
+	                      HANDLE_ATTR_NS_MASK;
```

Comment:
Was this mask added by accident?

I can't see how it could be needed.

Reply:
I copy-pasted this from our secure_read_service, which was updated by the TF-M maintainers.
Having them be equal in implementation makes sense to me. It may not be needed, but it doesn't hurt either.

Reply:
It's definitely not need, but I guess it does make it easier to compare the two implementations.